homeassistant/components/fyta/sensor.py

```diff
@@ -0,0 +1,171 @@
+        translation_key="fyta_scientific_name",
+    ),
+    FytaSensorEntityDescription(
+        key="plant_status",
+        translation_key="fyta_plant_status",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:flower",
+    ),
+    FytaSensorEntityDescription(
+        key="temperature_status",
+        translation_key="fyta_temperature_status",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer-lines",
+    ),
+    FytaSensorEntityDescription(
+        key="light_status",
+        translation_key="fyta_light_status",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:sun-clock-outline",
+    ),
+    FytaSensorEntityDescription(
+        key="moisture_status",
+        translation_key="fyta_moisture_status",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:water-percent-alert",
+    ),
+    FytaSensorEntityDescription(
+        key="salinity_status",
+        translation_key="fyta_salinity_status",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:sprout-outline",
+    ),
+    FytaSensorEntityDescription(
+        key="temperature",
+        translation_key="fyta_temperature",
+        native_unit_of_measurement="°C",
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:thermometer",
+    ),
+    FytaSensorEntityDescription(
+        key="light",
+        translation_key="fyta_light",
+        native_unit_of_measurement="mol/d",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:weather-sunny",
+    ),
+    FytaSensorEntityDescription(
+        key="moisture",
+        translation_key="fyta_moisture",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.MOISTURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:water-percent",
+    ),
+    FytaSensorEntityDescription(
+        key="salinity",
+        translation_key="fyta_salinity",
+        native_unit_of_measurement="mS/cm",
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:sprout-outline",
+    ),
+    FytaSensorEntityDescription(
+        key="ph",
+        translation_key="fyta_PH",
+        device_class=SensorDeviceClass.PH,
+        state_class=SensorStateClass.MEASUREMENT,
+        icon="mdi:ph",
+    ),
+    FytaSensorEntityDescription(
+        key="battery_level",
+        translation_key="fyta_battery_level",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.BATTERY,
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        icon="mdi:battery",
+    ),
+    FytaSensorEntityDescription(
+        key="last_updated",
+        translation_key="fyta_last_updated",
+        device_class=SensorDeviceClass.TIMESTAMP,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        icon="mdi:update",
+    ),
```

Comment:
Please leave this one for a followup, as I'm not 100% sure we like this kind of sensors. This way the PR can be merged faster

Reply:
This sensor holds the timestamp, the last time the Fyta Beam sensor sent data to the Hub (and the server, respectively). Insofar I think this has some relevance. Should I rename the sensor then (e.g. to last_sensor_update)?

Reply:
I have seen discussions in the past that this is a kind of sensor we don't want in HA, as HA also tracks the last moment a state got updated. I can't remember the outcomes and where to find the discussions, hence I want to propose to split it off to have this discussion in a separate PR so this discussion doesn't stop this PR from finishing.